graylog2-web-interface/src/components/bootstrap/BootstrapModalWrapper.jsx

```diff
@@ -42,7 +42,7 @@ class BootstrapModalWrapper extends React.Component {
 
   render() {
     return (
-      <Modal show={this.state.showModal} onHide={this.hide}>
+      <Modal show={this.state.showModal} onHide={this.hide} bsSize="large">
```

Comment:
We should probably add a prop to control that `bsSize`, and I would also add it to `BootstrapModalForm`. In that way we can customize this when we need to.